src/classes/UTIL_Describe_TEST.cls

```diff
@@ -101,6 +101,35 @@ public with sharing class UTIL_Describe_TEST {
         try {
             picklistEntries = UTIL_Describe_API.getPicklistActiveValuesMap('Account', 'id');
         } catch(UTIL_Describe.SchemaDescribeException sde){}
-    }       
+    } 
 
+    // Test for childRelationshipNamesService functionality
+    @isTest
+    private static void childRelationshipNamesServiceTest() {
+
+        // Insert Account
+        Account adminAccount = UTIL_UnitTestData_API.getMultipleTestAccounts(1, UTIL_Describe_API.getAdminAccRecTypeID())[0];
+        adminAccount.Name = 'Test Account 000';
+        insert adminAccount;
+
+        Test.startTest();
+            Schema.SObjectType sObjectTypeAcc = adminAccount.getSObjectType();
+            List<String> allChildRelationshipNames = UTIL_Describe_API.getAllChildRelationshipNames(sObjectTypeAcc);
+        Test.stopTest();
+
+        System.assert(allChildRelationshipNames != NULL);
```

Comment:
@skosgi1 Consider the readability of `System.assertNotEquals(NULL, allChildRelationshipNames)`. In many cases when we are testing for an expected or unexpected value, System.assertEquals and System.assertNotEquals may be good candidates for making the code readable.